packages/components/src/FieldBase/FieldBase.test.tsx

```diff
@@ -101,28 +103,27 @@ test('render Field with label and helptext', () => {
   expect(error).not.toBeInTheDocument();
 });
 
-test('render Field with label and errorMessage', () => {
+test('render Field with label and errorMessage', async () => {
```

Comment:
Why async needed? I would it expect when `await` is somewhere used.

Reply:
true :D